### ISSUE TITLE
auto loading video not working with pcm audio

### ISSUE THREAD

**Issue by duck7000** [maxrd2/subtitlecomposer#128]
Auto loading video does not work when video contains pcm audio track.

After loading the subtitle the video is not present and does not play, but if you manually open the corresponding video it works like it should.

It only happens with video that contains pcm audio, all others auto load fine

**Comment by maxrd2** [comment]
So... i really can't reproduce this.  have tried it with file w/ PCM audio that didn't generate waveform before.
Can you give me all steps you do after starting Subtitle Composer from shell, and paste the output too.
Also I need to know what video player backend you are using.
Double check that auto load setting is on in preferences.
Thank you.

**Comment by duck7000** [comment]
Oke here it is

I double click on a subtitle file (srt in this case) with in the same directory the movie with exactly the same name (this is the movie that previous didn't produce a waveform)
SC opens and load the subtitle, but the player window stays empty as is the waveform. Play button is grayed out.
I use gstreamer as backend (mpv used to crash a lot in the past, that may be solved).
I changed the backend to mpv but that makes no difference.

If i do this procedure with every other file (that don't have pcm audio) it works and the video plays automatically.

Since you fixed the waveform for this video this is a issue, before that the video did play but no waveform.

If i load the video manually the video does play incl the waveform.

I just tried a different video that also has pcm audio and that one works perfectly.. so there is something with that specific video.
The ffmpeg output is in the closed thread about the waveform

I will try it from the terminal, load the subtitle and see if that makes any difference
